pkg/api/handlers/libpod/pods.go

```diff
@@ -554,10 +562,35 @@ func PodStats(w http.ResponseWriter, r *http.Request) {
 			utils.Error(w, http.StatusNotFound, err)
 			return
 		}
-
 		utils.InternalServerError(w, err)
 		return
 	}
 
-	utils.WriteResponse(w, http.StatusOK, reports)
+	w.Header().Set("Content-Type", "application/json")
+	coder := json.NewEncoder(w)
+	coder.SetEscapeHTML(true)
+
+	if err := coder.Encode(reports); err != nil {
+		logrus.Infof("Error from %s %q : %v", r.Method, r.URL, err)
+	}
+	flush()
+	if query.Stream {
+		for {
+			select {
+			case <-r.Context().Done():
+				return
+			default:
+				time.Sleep(time.Duration(query.Delay) * time.Second)
```

Comment:
so is delay just idling? wouldn't it make more sense to add a flag like `Since`? or `Until`? like our other logging functions have?

Reply:
This is more or less following the implementation you have in pods top.

Reply:
I think it makes more sense to follow this implementation as it's more or less similar use case to top.

Reply:
ok @jmguzik I am just pointing out that `pod top` and `pod stats` are different commands and usually when a flag is added remotely it needs to be added to the non-remote podman as well.

Reply:
Ah, it exists already. It even works. So, it's only making it public here.